server/routes/routerRequestHandlers/retrieveEducation.test.ts

```diff
@@ -0,0 +1,105 @@
+import type { EducationDto } from 'inductionDto'
+import { Request, Response } from 'express'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
+import retrieveEducation from './retrieveEducation'
+import aValidEducationResponse from '../../testsupport/educationResponseTestDataBuilder'
+
+jest.mock('../../services/educationAndWorkPlanService')
+describe('retrieveEducation', () => {
+  const educationAndWorkPlanService = new EducationAndWorkPlanService(
+    null,
+    null,
+  ) as jest.Mocked<EducationAndWorkPlanService>
+  const requestHandler = retrieveEducation(educationAndWorkPlanService)
+
+  const prisonNumber = 'A1234BC'
+  const userToken = 'a-user-token'
+
+  let req: Request
+  let res: Response
+  const next = jest.fn()
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    req = {
+      user: {
+        token: userToken,
+      },
+      params: { prisonNumber },
+    } as unknown as Request
+    res = {
+      locals: {},
+    } as unknown as Response
+  })
+
+  it('should retrieve Education and store on res.locals', async () => {
+    // Given
+    const educationDto = aValidEducationResponse()
```

Comment:
Almost right 😁  .... the service method returns the DTO, not the `EducationResponse`, so you'll need to change this line to something like 
```
const educationDto = aValidEducationDto()
```
and create a suitable test data builder for it 👍